src/select.jsx

```diff
@@ -246,7 +247,7 @@ var classBase = React.createClass({
     }, function () {
       this.onChange();
 
-      React.findDOMNode(this.refs['currentOption']).focus(); //eslint-disable-line dot-notation
+      ReactDOM.findDOMNode(this.refs['currentOption']).focus(); //eslint-disable-line dot-notation
```

Comment:
You don't need `findDOMNode` if you're using a `ref` that's an element rather than a component. With 0.14 the dom node is exposed directly, so you can just do `this.refs['currentOption'].focus()`